Venmo/src/main/java/com/braintreepayments/api/VenmoAccountNonce.java

```diff
@@ -51,14 +62,70 @@ static VenmoAccountNonce fromJSON(JSONObject inputJson) throws JSONException {
             username = details.getString(VENMO_USERNAME_KEY);
         }
 
-        return new VenmoAccountNonce(nonce, username, isDefault);
+        return new VenmoAccountNonce(nonce, username, isDefault, json);
+    }
+
+    VenmoAccountNonce(String nonce, String username, boolean isDefault, JSONObject json) throws JSONException {
+        super(nonce, isDefault);
+        this.username = username;
+
+        if (json.has(VENMO_PAYMENT_METHOD_ID_KEY)) {
+            if (json.has(VENMO_PAYER_INFO_KEY)) {
```

Comment:
Based on the fixture added, it looks like these two keys are on the same level in the JSON - do we need both checks?  If we do - can it be `json.has(VENMO_PAYMENT_METHOD_ID_KEY) && json.has(VENMO_PAYER_INFO_KEY)` rather than a nested if statement?